app/frontend/src/features/dashboard/constants.ts

```diff
@@ -44,8 +44,8 @@ export const WEEKLY_EVENTS_MARKDOWN = `We have two weekly online events and **yo
 
 The events are held every week on:
 
-* Sundays at 19:00 UTC ([click here for your local time](https://mytime.io/19:00/UTC)), and
-* Tuesdays at 11:00 UTC ([click here for your local time](https://mytime.io/11:00/UTC)).
+* Tuesdays at 11:00 UTC ([click here for details and your local time](https://community.couchers.org/t/weekly-online-event-tuesday/1098)), and
+* Sundays at 19:00 UTC ([click here for details and your local time](https://community.couchers.org/t/weekly-online-event-sunday/684)).
```

Comment:
Why swap?

Reply:
Tuesday comes before Sunday in international convention